examples/open-ai-plugin/packages/service/src/todo.ts

```diff
@@ -0,0 +1,94 @@
+import { extendApi } from "@anatine/zod-openapi";
+import { command, entity } from "@eventual/core";
+import { ulid } from "ulidx";
+import z from "zod";
+
+const userNameSchema = extendApi(z.string(), {
```

Comment:
I really don't like the extend api ergonomics. Module augmentation of zod would be better imo.

Reply:
`@anatine/zod-openapi` only supports this method.

There was another zod to opanapi library that supported module augmentation, but the inclusion of a side effect statement would be more complex than a single import and wrapping the types.

Reply:
https://github.com/anatine/zod-plugins/issues/82